### PR TITLE
refresh "about" page and add tarball warnings near dl link

### PR DESCRIPTION
**Pull request by nicolasochem**

### CODE REVIEW

File: getting-started.md [hunk 1]
```diff
@@ -25,82 +25,103 @@ page_nav:
...
 This website helps people operating Tezos nodes to synchronize to the head of the chain, so they are operational faster.
 
-New snapshots are uploaded twice a day. (Granada and Mainnet)
+## What is a snapshot?
 
-We're are migrating to a new snapshot system where -
+A Tezos snapshot is a package containing Tezos chain data necessary for the node to synchronize to the network and validate blocks.
 
-Hangzhounet (currently deployed)
+In the absence of a snapshot, running a Tezos node for the first time requires syncing the historical mainnet chain from its genesis in June 2018, which takes several days. Consequently, most people spinning up new nodes use a snapshot.
```
Comment (orcutt989) [inline getting-started.md]:
```suggestion
In the absence of a snapshot, running a Tezos node for the first time requires syncing the historical mainnet chain from its genesis on June 2018, which takes several days. Consequently, most people spinning up new nodes use a snapshot.
```

File: getting-started.md [hunk 2]
```diff
@@ -25,82 +25,103 @@ page_nav:
...
 
-* Rolling snapshots **every 35 minutes**
+Importing a Tezos snapshot performs a sanity check of its contents. Consequently, it still takes a bit of time to import a snapshot: currently up to one hour on mainnet.
```
Comment (orcutt989) [inline getting-started.md]:
```suggestion
Importing a Tezos snapshot performs a sanity check of its contents. Consequently, it still takes a bit of time to import a snapshot- currently up to one hour on mainnet.
```

File: getting-started.md [hunk 3]
```diff
@@ -25,82 +25,103 @@ page_nav:
...
 
-Granada (planned)
+We provide **permalinks**: URLs that never change and reliably point to a recent snapshot. This allows to write automation where your node may automatically downlad a recent snapshot at first boot.
 
-* Rolling snapshots **every 35 minutes**
+For example, to download a recent full snapshot of Tezos mainnet, simply do:
 
-* Rolling tarballs **every 35 minutes**
+```bash
+wget https://mainnet.xtz-shots.io/full
```
Comment (orcutt989) [inline getting-started.md]: We don't provide full snapshots yet so do we want to say this now?

File: getting-started.md [hunk 4]
```diff
@@ -25,82 +25,103 @@ page_nav:
...
 
-This project was sponsored by a grant from the [Tezos foundation](https://tezos.foundation/).
+### Rolling tarballs
 
-## What is a tarball ?
+Rolling tarballs are generated from our rolling snapshots. We take a rolling snapshot, import it into a node, then package its data directory into a tarball.
+
+Effectively, by importing a tarball instead of a snapshot, you skip the snapshot import that the node would normally do.
+
+Consequently, by importing a tarball, you can go from zero to a fully synced node in less time: cyrrently it takes about **9 minutes** to get a fully synced rolling node.
```
Comment (orcutt989) [inline getting-started.md]:
```suggestion
Consequently, by importing a tarball, you can go from zero to a fully synced node in less time: currently it takes about **9 minutes** to get a fully synced rolling node.
```

File: getting-started.md [hunk 5]
```diff
@@ -25,82 +25,103 @@ page_nav:
...
+
+Archive tarballs contain the full backend data directory of an archive node, complete will all history of transactions since genesis. Archive snapshots do not exist, so importing a tarball is the only way of getting an archive node, short of syncing from scratch.
+
+Most Tezos users do not need an archive node. It is mostly useful for indexing the entire chain.
 
-Tezos snapshots can take a long time to import, especially for mainnet. 
+### Internal sanity checks
 
-However, we simply tar up `/var/node/tezos` on our rolling and archive nodes (excluding `peers.json` and `identy.json`), lz4 it, and provide it for you to download.
+Before making tarballs available, we test them internally: we import them and makes sure that the node starts and RPC becomes available.
```
Comment (orcutt989) [inline getting-started.md]:
```suggestion
Before making tarballs available, we test them internally: we import them and make sure that the node starts and RPC becomes available.
```

File: getting-started.md [hunk 6]
```diff
@@ -25,82 +25,103 @@ page_nav:
...
+
+For mainnet:
+
+* Rolling snapshots **every 1 hour and 45 minutes**
+
+* Rolling tarballs **every 1 hour and 45 minutes**
+
+* Archive tarballs **every 12 hours**
 
-We display the Tezos node version used for snapshot generation. For best results, use the same version to import your snapshot. The snapshot format may have changed.
```
Comment (orcutt989) [inline getting-started.md]:
```suggestion
```
There are 2 blank lines here only need 1.
https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md012---multiple-consecutive-blank-lines

File: getting-started.md [hunk 7]
```diff
@@ -25,82 +25,103 @@ page_nav:
...
+Importing a Tezos snapshot performs a sanity check of its contents. Consequently, it still takes a bit of time to import a snapshot: currently up to one hour on mainnet.
 
-* Rolling tarballs **every 35 minutes**
+[Read more on the Tezos documentation](https://tezos.gitlab.io/user/snapshots.html)
 
-* Archive tarballs **every 35 minutes**
+### How to use
 
-Granada (planned)
+We provide **permalinks**: URLs that never change and reliably point to a recent snapshot. This allows to write automation where your node may automatically downlad a recent snapshot at first boot.
```
Comment (orcutt989) [inline getting-started.md]:
```suggestion
We provide **permalinks**: URLs that never change and reliably point to a recent snapshot. This allows to write automation where your node may automatically download a recent snapshot at first boot.
```

File: getting-started.md [hunk 8]
```diff
@@ -25,82 +25,103 @@ page_nav:
...
 
-However, we simply tar up `/var/node/tezos` on our rolling and archive nodes (excluding `peers.json` and `identy.json`), lz4 it, and provide it for you to download.
+Before making tarballs available, we test them internally: we import them and makes sure that the node starts and RPC becomes available.
 
-For example- you will have a fully synced mainnet rolling node in **9 minutes**.
+### Caveats
+
+Snapshot import performs some safety checks to ensure that the chain state that you are importing is consistent. In contrast, importing a tarball completely bypasses these checks. As a consequence, by importing a tarball, you entrust the tarball provider to give you the correct mainnet chain.
+
+Please evaluate whether this is appropriate for your use case. If not, you may elect to use a snpashot or generate your own tarballs.
```
Comment (orcutt989) [inline getting-started.md]:
```suggestion
Please evaluate whether this is appropriate for your use case. If not, you may elect to use a snapashot or generate your own tarballs.
```

File: getting-started.md (hunk 3 above)
Comment (orcutt989) [reply getting-started.md]: Sorry this was a question not a commit.

Comment (nicolasochem) [reply getting-started.md]: right, I changed it to rolling

Comment (orcutt989) [reply getting-started.md]: Oop my bad! Thanks!!

**Pull request merged**